### PR TITLE
Finish show, filter words

### PR DESCRIPTION
**Pull request by minh93**

### CODE REVIEW

File: app/helpers/words_helper.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+module WordsHelper
+end
```
Comment (phuongbk) [inline app/helpers/words_helper.rb]: delete

File: app/controllers/words_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    case @filter_state
+    when "learned"
+      @words = @category.words.learned current_user
+    when "not_learn"
+      @words = @category.words.not_learn current_user, @category
+    else
+      @words = @category.words
+    end
+
+    @words = @words.paginate(page: params[:page], :per_page => 50)
```
Comment (biznezo) [inline app/controllers/words_controller.rb]: indent

File: app/helpers/words_helper.rb (hunk 1 above)
Comment (biznezo) [inline app/helpers/words_helper.rb]: remove this file

File: app/views/words/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,28 @@
+<div class="row">
+  <%= form_tag category_words_path(@category.id),
+    method: :get, class: "form-inline", id: "filter-form" do %>
+    <div class="form-group">
+      <label>
+        <%= select_tag :id, options_for_select(@categories_list, category_words_path(@category.id)), class: "form-control" %>
```
Comment (biznezo) [inline app/views/words/index.html.erb]: 2 lines

File: app/controllers/words_controller.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    when "learned"
+      @words = @category.words.learned current_user
+    when "not_learn"
+      @words = @category.words.not_learn current_user, @category
+    else
+      @words = @category.words
+    end
+
+    @words = @words.paginate page: params[:page], :per_page => 50
+
```
Comment (tunv) [inline app/controllers/words_controller.rb]: ident!

File: app/controllers/words_controller.rb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+class WordsController < ApplicationController
+  def index
+    @category = Category.find params[:category_id]
+    @filter_state = params[:filter_state] || Settings.category_type.all_word
+    @words = @category.words.send(@filter_state, current_user, @category)
```
Comment (yuyuvn) [inline app/controllers/words_controller.rb]: remove ()

**Pull request merged**